pkg/middleware/middleware.go

```diff
@@ -72,6 +87,21 @@ func Process(ctx *quaycontext.QuayRegistryContext, quay *v1.QuayRegistry, obj cl
 		return dep, nil
 	}
 
+	// If the current object is a PVC, check for volume override
+	if pvc, ok := obj.(*corev1.PersistentVolumeClaim); ok {
+		var volumeSize resource.Quantity
+		switch quayComponentLabel {
+		case "postgres":
+			volumeSize = getVolumeSizeOverrideForComponent(quay, v1.ComponentPostgres)
+		case "clair-postgres":
+			volumeSize = getVolumeSizeOverrideForComponent(quay, v1.ComponentClair)
+		case "objectstorage":
```

Comment:
I can't see this working. Do we create a persistent volume claim for the object storage? I thought we were only creating an ObjectBucketClaim. Can you clear this for me?

Reply:
@jonathankingfc I still don't understand this. What is this `case` supposed to do?

Reply:
This switches on the label taken from the current resource in the middleware pipeline. We could move this logic into the getVolumeSizeOverride() function, though

Reply:
My question is because I don't think we have a PVC with such label ("objectstorage"), hence my confusion I was wishing to get straight. Who creates this PVC ? As far as I remember we only have two PVCs (quay and clair databases).

Reply:
I've seen mentions to this component in the logs. From what I gather the operator doesn't currently have the option of managing it, but it's still treated as a component.
@jonathankingfc please confirm.

Reply:
@flavianmissi since the PVC for objectstorage is created manually by the user during the pre-install process (we want to keep this behavior), we will not be touching the object storage PVC. I have removed this line from the PR